test/test_overrides.py

```diff
@@ -621,6 +622,9 @@ def instance_gen():
                     func_args.append(torch.float32)
                 elif t == 'c10::string_view':
                     func_args.append('')
+                elif t == 'SymInt':
+                    # TODO: generate actual SymbolicInt
+                    func_args.append(1)
```

Comment:
This is fine, SymInt will accept regular int too. (cc @Krovatkin, but no action necessary)